lighthouse-core/gather/gatherers/devtools-log.js

```diff
@@ -42,16 +40,16 @@ class DevtoolsLog extends FRGatherer {
     this._messageLog.reset();
     this._messageLog.beginRecording();
 
-    this._networkMonitor = new NetworkMonitor(driver.defaultSession);
-    this._networkMonitor.on('protocolmessage', this._onProtocolMessage);
-    this._networkMonitor.enable();
+    driver.targetManager.on('protocolevent', this._onProtocolMessage);
+    await driver.defaultSession.sendCommand('Page.enable');
```

Comment:
we have so many `Page.enable`s in LH that this has to be unnecessary, but this matches the behavior of the `this._networkMonitor.enable()` call